compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -36,7 +36,9 @@
     REDECLARED_BUILTIN_SYMBOL("redeclared.builtin.symbol"),
     UNDEFINED_SYMBOL("undefined.symbol"),
     UNDEFINED_FUNCTION("undefined.function"),
```

Comment:
Can we remove this from the code after fixing it in every place?

Reply:
We cannot remove that one because it is still used to identify the function invocations as shown below.

```
public function foo() returns string {
    return "";
}

public function main() {
    Foo(); // Compile error : undefined function 'Foo'
}
```